frontend/packages/schema-editor/src/components/TypesInspector/TypeItem.tsx

```diff
@@ -35,9 +43,7 @@ export const TypeItem = ({ uiSchemaNode, selected, setSelectedTypePointer }: Typ
         <div>
           <FileJsonIcon className={classes.typeIcon} />
         </div>
-        <span className={classes.typeName}>
-        {name}
-      </span>
+        <span className={classes.typeName}>{name}</span>
```

Comment:
I am not sure if we need `<Paragraph> `  here, but if so, then we can fix it in a seperate issue because it is not related to the target of this issue 😊, and i will reveret all the changes in the component `TypeItem`